dts/bindings/regulator/x-powers,axp192-regulator.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright (c), 2021 NXP
+# Copyright (c), 2023 Martin Kiepfer <m.kiepfer@teleschirm.org>
+# SPDX -License-Identifier: Apache-2.0
+
+description: |
+  AXP192 PMIC
+
+  The PMIC has two DCDC converters and two LDOs. All need to be defined as
+  children nodes.
+  For example:
+
+  pmic@34 {
+    reg = <0x341>;
+    ...
+    DCDC1 {
+      /* all properties for DCDC1 */
+    };
+    DCDC2 {
+      /* all properties for DCDC2 */
+    };
+    DCDC3 {
+      /* all properties for DCDC3 */
+    };
+    LDO1 {
+      /* all properties for LDO1 */
+    };
+    LDO2 {
+      /* all properties for LDO2 */
+    };
+    LDO3 {
+      /* all properties for LDO3 */
+    };
+  };
+
+compatible: "x-powers,axp192-regulator"
+
+include: base.yaml
+
+child-binding:
+  include:
+    - name: regulator.yaml
+      property-allowlist:
+        - regulator-always-on
+        - regulator-boot-on
+        - regulator-min-microvolt
+        - regulator-max-microvolt
+  
+  description: Voltage output of PMIC controller regulator
+  properties:
+
+    microvolt:
```

Comment:
could you use regulator-init-microvolt for this?

Reply:
Thanks for the hint. Resolved.